src/DoubleTheDonation/FormExtension/Block/Edit.tsx

```diff
@@ -0,0 +1,57 @@
+import {__} from '@wordpress/i18n';
+import {InspectorControls, useBlockProps} from '@wordpress/block-editor';
+import {PanelBody, PanelRow, TextareaControl, TextControl} from '@wordpress/components';
+import {NeedsSetup} from '../Template/Variations';
+
+declare const window: {
+    GiveDTD: {
```

Comment:
I'm still under the influence of the legacy code :smile:  - I'll update this, but in the last PR. I would like to avoid merge conflicts as much as possible